### DISCUSSION TITLE
Cannot serve in Windows 10

### DISCUSSION THREAD

**Discussion by auliabismar** [pocketbase/pocketbase#268] [Q&A]
Using PocketBase 0.3.4 in Windows 10 after using `--http="127.0.0.1:8090"` I get another error:
`listen tcp 127.0.0.1:8090: bind: An attempt was made to access a socket in a way forbidden by its access permissions.`
I tried restarting hns service and added the exe to windows firewall, but no luck,

Please advise

**Comment by ganigeorgiev** [comment]
Unfortunately, I have very little experience with Windows and I'm not sure that I'll be able to help you.

Googling the issue I stumbled on couple of things that you could check:

- ensure that there is no other service running on the 8090 port

- try with lower port, eg. 3000, because the configured Windows "dynamic ports" may have lower upper range, eg. 5000 (https://superuser.com/questions/1579346/many-excludedportranges-how-to-delete-hyper-v-is-disabled/1671710#answer-1671710)

- run `netsh interface ipv4 show excludedportrange protocol=tcp` to see the reserved ports - https://stackoverflow.com/questions/39783421/attempt-was-made-to-access-a-socket-in-a-way-forbidden-by-its-access-permissions#answer-63009050

- try restarting the winnat service - https://github.com/docker/for-win/issues/3171#issuecomment-739740248
